app/Http/Controllers/ForumController.php

```
@@ -215,12 +247,65 @@ public function destroy(ForumPost $post): RedirectResponse
     }
+
+    public function report(Request $request, ForumPost $post): JsonResponse
+    {
+        $user = $request->user();
+        abort_unless($user, 401, 'Unauthenticated');
+
+        $validated = $request->validate([
+            'reason' => ['required', 'string', 'max:255'],
+        ]);
+
+        $alreadyReported = Report::where('reporter_id', $user->id)
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Report::where().


(staticMethod.notFound)